src/Prefixer.js

```diff
@@ -66,6 +67,16 @@ export function prefix (value, length) {
 		// grid-template-(columns|rows)
 		case 2592: case 3360:
 			return MS + replace(value, 'template-', '') + value
+		// grid-(row|column)-start
+		case 4384: case 3616:
+			if (children = children && children.find(function (element) { return match(element.props, /grid-\w+-end/) })) {
```

Comment:
[IE11 doesn't support Array.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find). Would defeat the purpose to support IE11 grids with unsupported JS api's. Let's use `.some` and set the length variable when a match is found we use that to index into children i.e

```js
if (children.some(function (element, index) { return length = index, match(element.props, /grid-\w+-end/) })) {
```

Then below just:

```js
return ~indexof(value + (children = children[length].value), 'span')
```